### PR TITLE
Configure repo

### PR DESCRIPTION
**Pull request by mayaCostantini**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -4,4 +4,4 @@ aliases:
     - goern
     - harshad16
     - KPostOffice
-    - pacospace
+    - mayaCostantini
```
Comment (codificat) [inline OWNERS_ALIASES:7]: is this file actually used in this repo?

File: OWNERS [hunk 2]
```diff
@@ -3,7 +3,7 @@
 approvers:
   - goern
   - fridex
+  - mayacostantini
```
Comment (codificat) [inline OWNERS:6]: Offering myself to help maintain:

```suggestion
  - codificat
  - goern
  - fridex
  - mayacostantini
```

File: .aicoe-ci.yaml [hunk 3]
```diff
@@ -1,15 +1 @@
-# Setup and configuring aicoe-ci with configuration file `.aicoe-ci.yaml`
-# Example `.aicoe-ci.yaml` with a full list of config options is available here: https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml
----
```
Comment (codificat) [inline .aicoe-ci.yaml:3]: I would leave that comment there, it doesn't hurt to point to docs

**Review by codificat — COMMENTED** [review]
Adding some suggestions

**Comment by mayaCostantini** [comment]
Thanks @codificat for the suggestions :100: Do you also want to be a reviewer of this repo? 
And @thoth-station/devs you are welcome to add or remove yourself from the maintainers/approvers list :+1:

File: .thoth.yaml [hunk 4]
```diff
@@ -59,10 +37,15 @@ managers:
   - name: version
     configuration:
       # A list of maintainers (GitHub or GitLab accounts) of this repository:
-      maintainers: []
+      maintainers:
```
Comment (codificat) [inline .thoth.yaml:40]: I forgot to mention this before: let's remove the `maintainers` section from `.thoth.yaml`, as this repo already has an OWNERS file and Kebechet will take that list from the `approvers` in that file - so, removing this section from here avoids redundancy and potential inconsistencies

**Review by codificat — CHANGES_REQUESTED** [review]
One more thing about removing `maintainers` from `.thoth.yaml` (relying on OWNERS instead)

**Comment by mayaCostantini** [comment]
@codificat Ready for review :+1:

**Comment by codificat** [comment]
Thanks,
/lgtm

**Comment by mayaCostantini** [comment]
Merging as discussed with @codificat

**Pull request merged**